examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -901,6 +994,12 @@ impl FrontendNode {
         Ok(())
     }
 
+    fn test_channel_chain_leaked(&mut self) -> TestResult {
```

Comment:
Do the channels leak because they are not closed?

Reply:
Yep.  The handle value is still in the per-Node handle table managed by the Runtime, but if there's no memory of what the value is, it can't be accessed.  (Oak's handle values are randomized, unlike Unix file descriptors)